app/assets/stylesheets/earthworks.css

```diff
@@ -55,4 +55,17 @@
   --bs-btn-hover-border-color: var(--stanford-digital-red-dark)!important;
   --bs-btn-active-bg: var(--stanford-digital-red-darker)!important;
   --bs-btn-active-border-color: var(--stanford-digital-red-darker)!important;
+}
+
+.documents-list article:hover {
+  box-shadow: -4px 4px 4px 0px var(--stanford-fog-light);
+}
```

Comment:
this should be the `:focus` style too, right?

Reply:
That makes sense.